src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -310,9 +310,22 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		let defaultParameters = new Map<string, string>();
 		editor.cells.forEach(cell => {
 			if (cell.isParameter) {
+				if (!cell.source) {
+					// If there is no parameters in the cell indicate to user to add them
+					this.notificationService.notify({
+						severity: Severity.Info,
+						message: localize('noParametersInCell', "This notebook cannot run with parameters until a there are parameters added to the parameter cell. [Learn more](https://docs.microsoft.com/sql/azure-data-studio/notebooks/notebooks-parameterization)."),
+					});
+				}
 				for (let parameter of cell.source) {
-					let param = parameter.split('=', 2);
-					defaultParameters.set(param[0].trim(), param[1].trim());
+					if (parameter.includes('=')) {
+						// For syntax that includes multiple parameters on one line (ex. x=1;y=2)
+						let multiParams = parameter.split(';');
```

Comment:
And what if my parameter has a ; in it?

`var param1 = 'This will be parsed incorrectly; oh no!'

Reply:
discussed this will be documented that multi-line parameters are to not be typically used for parameters cell and therefore support for this would require a whole parsing/validation of the string which is something that will look into further for supporting multiple other languages for parameterization.